### ISSUE TITLE
add tan-1(arctan),sin-1(arsin),cos-1(arccos) features

### ISSUE THREAD

**Issue by AhmedAshrafHamza** [SimpleMobileTools/Simple-Calculator#110]
Scientific calculator does not include  tan-1(arctan),sin-1(arsin),cos-1(arccos) features

**Comment by Phoenix1747** [comment]
IMHO the trigonometric functions don't need to be in a simple calculator. At least not before there aren't any brackets.

**Comment by K4LCIFER** [comment]
> IMHO the trigonometric functions don't need to be in a simple calculator. At least not before there aren't any brackets.

I disagree. A trig function is very handy when doing basic carpentry for, example -- just a very quick angle calculation. One doesn't need brackets to quickly calculate an angle.

**Comment by ionuttbara** [comment]
+1
